remotespark/livyclientlib/livysession.py

```diff
@@ -12,7 +13,8 @@
 from .livyreliablehttpclient import LivyReliableHttpClient
 
 
-class LivySession(object):
+
+class LivySession(GuidMixin, object):
```

Comment:
How did you try it and had it work Mohamed? I'd be very interested in seeing how you got it to work without calling it explicitly.